config/olm/openshift/0.2.0/manifests/dynatrace-kubernetes-monitoring_rbac.authorization.k8s.io_v1_clusterrole.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: dynatrace-kubernetes-monitoring
```

Comment:
Why is it a different file instead of being on the CSV?

Reply:
Actually a good question - it got auto generated by running the make bundle command